apple-pie/index.html

```diff
@@ -4,18 +4,67 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- don't forget to link your styles -->
+    <link rel="stylesheet" href="styles/style.css">
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+    </style>
   </head>
   <body>
-    Apple Pies This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
-    be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
-    favorites as well! Ingredients 1 recipe pastry for a 9 inch double crust pie 1/2 cup unsalted butter 3 tablespoons
-    all-purpose flour 1/4 cup water 3 tablespoons all-purpose flour 1/4 cup water 1/2 cup white sugar 1/2 cup packed
-    brown sugar 8 Granny Smith apples - peeled, cored and sliced Directions Preheat oven to 425 degrees F (220 degrees
-    C). Melt the butter in a saucepan. Stir in flour to form a paste. Add water, white sugar and brown sugar, and bring
-    to a boil. Reduce temperature and let simmer. Place the bottom crust in your pan. Fill with apples, mounded
-    slightly. Cover with a lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-    that it does not run off. Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
-    degrees C). Continue baking for 35 to 45 minutes, until apples are soft.
+
+  <header>
+      <h1 class="main-title">
+        Apple Pie
+      </h1>
+  </header>
+    
+    <div class="page-margins">
+    
+      
+      
+      <p class="descriptive-text">
+        This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
+        be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
+        favorites as well!
+      </p> 
+
+      <img class="img-info" src="/Users/hevesygab/Documents/Documents/VS/Aula 1/lab-css-recipes-clone/apple-pie/images/recipe-info.png" alt="Recipe prep information">
```

Comment:
Here, in order to link images that are in your local computer you should do the path to the image as follows:
```
src="images/recipe-info.png"
```